modules/RTC/TPCUtils.js

```diff
@@ -34,19 +35,93 @@ export class TPCUtils {
      */
     constructor(peerconnection) {
         this.pc = peerconnection;
-        const bitrateSettings = this.pc.options?.videoQuality?.maxBitratesVideo;
-        const standardBitrates = {
-            low: LD_BITRATE,
-            standard: SD_BITRATE,
-            high: HD_BITRATE,
-            ssHigh: HD_BITRATE
+        this.codecSettings = STANDARD_CODEC_SETTINGS;
```

Comment:
We need to clone the object here. Just by using the reference we'll be mutating the STANDARD_CODEC_SETTINGS object.